london_housing/Unit_4_Challenge_Tier_3.ipynb

```diff
@@ -0,0 +1,4406 @@
+{
```

Comment:
<p>Nice observation that you can slice the 'non borough' entries by index. I often use pandas to produce lists like this, but will ultimately create a list on entries so that it is more readable to the next person.</p><p> </p><p>Example:</p><pre class="ql-syntax" spellcheck="false"># non of these are boroughs of london
non_london_boroughs = ['Inner London', 'Outer London', 'NORTH EAST',
'NORTH WEST', 'YORKS &amp; THE HUMBER', 'EAST MIDLANDS', 'WEST MIDLANDS',
'EAST OF ENGLAND', 'LONDON', 'SOUTH EAST', 'SOUTH WEST', 'England',
'City of London'] 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Kazage/Data-Science-Notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>